looprpc/client.proto

```diff
@@ -1440,3 +1478,220 @@ message InstantOut {
     */
     string sweep_tx_id = 5;
 }
+
+message NewStaticAddressRequest {
+    /*
+    The client's public key for the 2-of-2 MuSig2 taproot static address.
+    */
+    bytes client_key = 1;
```

Comment:
This field is not passed by `loop` where this API is called.
If the field is optional, I propose to add this information to its description.
Otherwise it might be a bug in `cmd/loop/staticaddr.go` that it is not passed there.